README.md

```diff
@@ -61,11 +61,15 @@ console.log(`The header is: ${header}`)
 
 This outputs `The header is: ENTER HEADER CODE HERE`. Do you see what happened there? The `${header}` took the contents of `header` and made sure they were sent to the console. 
 
-Our task in this lab is to modify the three variables to be the correct `querySelector` to select the correct elements. 
+Our task in this lab is to modify the three variables to be the correct `querySelector` to select the correct elements. Then the `console.log` statements will output the code we wrote. 
```

Comment:
tell the students what the three variables are. Header, firstLinkText and firstLinkHREF